pyproject.toml

```diff
@@ -12,11 +12,12 @@ build-backend = "setuptools.build_meta"
 
 [tool.cibuildwheel]
 archs = ["auto64"]
-test-extras = ["test", "all"]
+test-requires = ["scipy", "graphviz", "pooch"]
+test-extras = ["test"]
```

Comment:
What is the difference between `test-requires` and `test-extras`?

Reply:
"extras" refers to the package "extras" that will be installed, i.e., it will `pip install scipp[test]`. "requires" is just a list of extra requirements.

Reply:
So with this change, the test extras will only be installed with `pip install scipp[test]` and not `pip install scipp[all]`? And it will install scipy, graphviz, and pooch in that case?

Reply:
This change has no effect on `scipp[all]`. `scipp[all]` includes `h5py` which does not work on Python 3.11, so we are not installing this in the wheel build any more. Instead I am listing the *other* test dependencies by hand.